src/containers/SocialPage/IntroBlock.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { Link } from 'react-router'
+import instagram_logo from '../../assets/instagram_logo.png'
+import facebook_logo from '../../assets/facebook_logo.png'
+import snapchat_logo from '../../assets/snapchat_logo.png'
+import twitter_logo from '../../assets/twitter_logo.png'
+
+class IntroBlock extends React.Component {
+
+  static propTypes = {
+    data: React.PropTypes.object,
+  }
+  
+  render () {
+
+    return (
+      <div className="db dt-ns mw9 center">
+        <div className="fl-m fl-l w-50-m w-50-l">
+          <div className="pa4 pa4-m">
+          <h2 className="f2">Facebook</h2>
+            <img className="logo" src={facebook_logo}/>
+            <p className="mid-gray f4 lh-copy">            
+              Follow us to get the latest news @farnboroughguide
+            </p> 
+          </div>
+        </div>
+        <div className="fl-m fl-l w-50-m w-50-l">
+          <div className="pa4 pa4-m">
+            <h2 className="f2">Instagram</h2>
+            <div clasaName="size">
```

Comment:
looks like there may be two occurrences of this??? Just do a search and replace "clasaName" for "className" that will fish them all out